src/common/functions.js

```diff
@@ -0,0 +1,117 @@
+export const s4 = () => {
+  return Math.floor((1 + Math.random()) * 0x10000)
+    .toString(16)
+    .substring(1);
+};
+
+// generates uniqueID
+export const uniqueId = () => {
+  return s4() + s4() + '-' + s4() + '-' + s4() + '-' + s4();
+};
```

Comment:
Group this function with the s4() function in its own file to separate functionality